cloudinit/net/__init__.py

```diff
@@ -486,7 +486,17 @@ def find_candidate_nics_on_linux() -> List[str]:
                 "Found unstable nic names: %s; calling udevadm settle",
                 unstable,
             )
-            util.udevadm_settle()
+            try:
+                util.udevadm_settle()
+            except subp.ProcessExecutionError as error:
+                LOG.error(
```

Comment:
In some cases this is not a failure, just a potential cause of failure. Logging at a warning level might be better to indicate that distinction.

Reply:
makes sense